go.mod

```diff
@@ -205,3 +205,4 @@ replace google.golang.org/grpc v1.54.0 => google.golang.org/grpc v1.26.0
 // kvproto at the same time. You can run `go mod tidy` to make it replaced with go-mod style specification.
 // After the PR to kvproto is merged, remember to comment this out and run `go mod tidy`.
 // replace github.com/pingcap/kvproto => github.com/$YourPrivateRepo $YourPrivateBranch
+replace github.com/pingcap/kvproto => github.com/HuSharp/kvproto v0.0.0-20230710073438-f629a3cd43e9
```

Comment:
remove this patch and update the version

Reply:
Okay, when TiKV and PD all get merge label, I will update kvproto.